repose-aggregator/src/docs/asciidoc/architecture/container.adoc

```diff
@@ -107,6 +114,13 @@ If all of the follow conditions are met, *Repose* may delete deployment director
 To avoid this situation, it is highly recommended that the same `deployment-directory` is not used for more than one *Repose* instance.
 ====
 
+[WARNING]
+====
+Files in the deployment directory are only removed if the `auto-clean` feature is used, and even then, only when *Repose* shuts down.
+Since files in the deployment directory are made accessible to *Repose* at runtime (via the classpath), files in the deployment directory that are not managed by *Repose* could affect the behavior of *Repose* and could even result in errors.
+As such, care should be taken to ensure that errant files do not end up in the deployment directory.
+====
+
```

Comment:
I think that note has already been covered. Check out the `WARNING` above this one.

Reply:
Heh, I would still recommend not using the same deployment directory because I would recommend not running multiple instances of Repose on the same host.

But I see your point. I'll update that warning to recommend not using the same deployment directory alongside auto-clean.

Reply:
Well now that is a whole other can of worms.
I do see those docs coming up shortly too.

- Don't route to clusters.
- Just Don't use clusters at all.
- Don't run multiple Repose instances on a single host.

Reply:
Truth.

Anyway, see if those new words are better.

Reply:
:+1: